### ISSUE TITLE
0.2.1-1 seems broken

### ISSUE THREAD

**Issue by Xuanwo** [mlua-rs/luarocks-build-rust-mlua#12]
Hi, thanks for maintaining this project. I found the newly released `0.2.1-1` seems to be broken on our build. Would you like to take a look? Thanks.

https://github.com/apache/opendal/actions/runs/10197254847/job/28212887362

```shell
  sudo "PATH=$PATH" luarocks make
  busted test/opendal_test.lua
  shell: /usr/bin/bash -e {0}
  env:
    RUSTFLAGS: -C force-frame-pointers=yes -C debuginfo=1
    RUST_BACKTRACE: 1
    RUST_LOG: opendal=trace
    CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
Missing dependencies for opendal 0.1.3-1:
   luarocks-build-rust-mlua (not installed)

opendal 0.1.3-1 depends on lua >= 5.1 (5.2-1 provided by VM)
opendal 0.1.3-1 depends on luarocks-build-rust-mlua (not installed)
Installing https://luarocks.org/luarocks-build-rust-mlua-0.2.1-1.src.rock

No existing manifest. Attempting to rebuild...
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `/root/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustc - --crate-name ___ --print=file-names --target target --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=split-debuginfo --print=crate-name --print=cfg` (exit status: 1)
  --- stderr
  error: Error loading target specification: Could not find specification for target "target". Run `rustc --print target-list` for a list of built-in targets


Error: Build error: Failed building.
luarocks-build-rust-mlua 0.2.1-1 is now installed in /usr/local (license: MIT)
```

**Comment by Xuanwo** [comment]
I'm guessing it's related to https://github.com/mlua-rs/luarocks-build-rust-mlua/commit/44eb3ae7fbf225994948793e438471fb2a03ba69

**Comment by khvzak** [comment]
Thanks for the report, I deleted this version until further investigation!

**Comment by khvzak** [comment]
I published v0.2.2 that should have the issue solved

**Comment by Xuanwo** [comment]
Thank you @khvzak for fix this, closing now.
